frontend/src/app/api/auth/[...nextauth]/route.js

```diff
@@ -34,16 +34,16 @@ const handler = NextAuth({
       };
 
       try {
-        const res = await axios.post(
-          `${process.env.EXPRESS_SERVER}/auth/signin-new`,
-          { user: user_ }
-        );
-        if (res.status === 200) {
-          // Successful request
+        const res = await axios.post(`${process.env.EXPRESS_SERVER}/auth/signin-new`,
+            {
+              name: user_.name,
+              email: user_.email,
+              image: user_.image
+            });
+        if (res.status === 200) { // Successful request
```

Comment:
Changed request body to pass in information individually (as backend had issues with deconstructing the json object)